src/stylesheets/elements/typography/_prose.scss

```diff
@@ -1,6 +1,10 @@
 .usa-prose {
   @include typeset($theme-prose-font-family);
+
   & > {
-    @include usa-content-styles;
+    @include usa-paragraph-style;
+    @include usa-headings-styles;
+    @include usa-list-styles;
+    @include usa-table-styles;
```

Comment:
@thisisdano you may want to remove the link style mixin from usa-content-styles and then re-add it here.

Reply:
I think I will leave link styles in `usa-content-styles` if only because link styles are currently included when `$theme-global-content-styles: true` and I want to keep that conceptually consistent (ie `$theme-global-content-styles` means use `usa-content-styles`)